app/code/Magento/Checkout/Model/ShippingInformationManagement.php

```diff
@@ -185,7 +185,7 @@ public function saveAddressInformation(
         } catch (\Exception $e) {
             $this->logger->critical($e);
             throw new InputException(
-                __('The shipping information was unable to be saved. Verify the input data and try again.')
+                __('The shipping information was unable to be saved. Error: "%1"', $e->getMessage())
```

Comment:
Additionally - placeholder is recommended:
`Error: "%message"', ['message' => $e->getMessage()]`